actors/builtin/miner/miner_state.go

```diff
@@ -106,6 +106,8 @@ type MinerInfo struct {
 	// The number of sectors in each Window PoSt partition (proof).
 	// This is computed from the proof type and represented here redundantly.
 	WindowPoStPartitionSectors uint64
+
+	ConsensusFaultReported abi.ChainEpoch
```

Comment:
Please document this.

Please also add a `policy.go` parameter of `ConsensusFaultIneligibilityDuration = Finality` in preparation for #887.

Thinking about this yesterday, it might be simpler to set this epoch as the first epoch _after_ the penalty period, when they become eligible again (`ConsensusFaultElapsed`). That way, if we init to `-1`, there's no other tests or arithmetic necessary to determine whether a miner is currently ok. But your call.

Reply:
Yeah I like the first unpenalized epoch idea.

Reply:
Thanks for the comment, please rename it too